CODEOWNERS

```diff
@@ -1 +1,6 @@
-* @JoCa96 @BoppLi @larsrickert
\ No newline at end of file
+# Global code owners (developers)
+* @JoCa96 @BoppLi @larsrickert
+
+# Define designers as code owners for screenshots to automatically
+# request them as reviewer if visual changes exist
+/packages/sit-onyx/playwright/snapshots/ @jannick-ux
```

Comment:
will only UX be able to approve those PRs or we as well? should there be a policy that then requires 1 approval from each "team" or else requires a total of 2 approvals?

also, martin/till need to create their accounts so jannick is allowed to have vacation ever again :p

Reply:
Good point, I added martin :) Yes all reviewers can approve so as we currently only require 1 reviewer, either a dev or ux could approve in this case